sql-select/actor-forty.sql

```diff
@@ -0,0 +1,4 @@
+select *
+from "actors"
+where "actorId" = '40'
```

Comment:
The single quotes make this a string, which postgres then needs to convert to a number. Hopefully, it does it the way you intended. To remove the uncertainty, always use the proper data types. Numbers should be specified without quotes.